src/actions/integrations.js

```diff
@@ -378,6 +379,43 @@ export function getOAuthApp(appId: string) {
     });
 }
 
+export function getAuthorizedOAuthApps() {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        dispatch({type: IntegrationTypes.GET_AUTHORIZED_OAUTH_APPS_REQUEST, data: {}});
+
+        const state = getState();
+        const currentUserId = getCurrentUserId(state);
+
+        let data;
+        try {
+            data = await Client4.getAuthorizedOAuthApps(currentUserId);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+
+            dispatch(batchActions([
+                {type: IntegrationTypes.GET_AUTHORIZED_OAUTH_APPS_FAILURE, error},
+                logError(error),
+            ]), getState);
+
+            return {error};
+        }
+
+        dispatch({type: IntegrationTypes.GET_AUTHORIZED_OAUTH_APPS_SUCCESS, data: {}});
+
+        return {data};
+    };
+}
+
+export function deautorizeOAuthApp(clientId: string) {
```

Comment:
Spell correction to,
```suggestion
export function deauthorizeOAuthApp(clientId: string) {
```